generated/nidaqmx/constants.py

```diff
@@ -394,15 +401,15 @@ class InputCalSource(Enum):
 
 
 class InputDataTransferCondition(Enum):
-    ON_BOARD_MEMORY_MORE_THAN_HALF_FULL = 10237  #: Transfer data from the device when more than half of the onboard memory of the device fills.
-    ON_BOARD_MEMORY_NOT_EMPTY = 10241  #: Transfer data from the device when there is data in the onboard memory.
-    ONBOARD_MEMORY_CUSTOM_THRESHOLD = 12577  #: Transfer data from the device when the number of samples specified with **ai_data_xfer_custom_threshold** are in the device FIFO.
-    WHEN_ACQUISITION_COMPLETE = 12546  #: Transfer data when the acquisition is complete.
+    ONBRD_MEM_MORE_THAN_HALF_FULL = 10237  #: Transfer data from the device when more than half of the onboard memory of the device fills.
+    ONBRD_MEM_NOT_EMPTY = 10241  #: Transfer data from the device when there is data in the onboard memory.
+    ONBRD_MEM_CUSTOM_THRESHOLD = 12577  #: Transfer data from the device when the number of samples specified with **ai_data_xfer_custom_threshold** are in the device FIFO.
+    WHEN_ACQ_COMPLETE = 12546  #: Transfer data when the acquisition is complete.
```

Comment:
ONBOARD, MEMORY, and ACQUISITION please

Reply:
Updated enum values as in expected constants.py(`ON_BOARD_MEMORY_MORE_THAN_HALF_FULL,ON_BOARD_MEMORY_NOT_EMPTY,ONBOARD_MEMORY_CUSTOM_THRESHOLD,WHEN_ACQUISITION_COMPLETE`)